src/app.rs

```diff
@@ -183,9 +194,20 @@ impl<'a, 'b> Application<'a, 'b> {
         profile_scope!("dispatch");
         self.dispatcher.dispatch(&mut self.engine.world.res);
 
+        for local in &mut self.locals {
+            local.run_now(&self.engine.world.res);
+        }
+
         #[cfg(feature = "profiler")]
         profile_scope!("maintain");
         self.engine.world.maintain();
+
+        // TODO: replace
+        // TODO: let the user handle
```

Comment:
Could you add more text to these todos?  The task they describe isn't quite clear.

Reply:
Sure. It's basically about what we discussed via Gitter this morning: for some errors, users want the application to quit, others just print and pop the state, or print and continue, ...